src/cloudformation/service-elb.template

```diff
@@ -562,7 +562,6 @@
       "Type": "AWS::AutoScaling::AutoScalingGroup",
       "Properties": {
         "Cooldown": "300",
-        "DesiredCapacity": {"Ref": "InstanceMin"},
```

Comment:
Breadcrumb: the reason we're taking this out, is that currently an update to the AutoScalingGroup returns the `Desired` back to this number, even though it might have had legitimate reasons to upscale.
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-desiredcapacity
Docs point out that this is not a required property, so leaving it out should mean the `Desired` is retained.